dozer/cogs/management.py

```diff
@@ -0,0 +1,209 @@
+"""General, basic commands that are common for Discord bots"""
+
+import asyncio
+import json
+import logging
+import math
+import os
+
+from datetime import timezone, datetime
+
+import humanize
+import discord
+from dateutil import parser
+from discord.ext.commands import has_permissions, CommandInvokeError
+
+from ..db import asyncpg
+from ._utils import *
+from .general import blurple
+from .. import db
+
+DOZER_LOGGER = logging.getLogger(__name__)
+
+timezone_file = "timezones.json"
+
+
+class Management(Cog):
+    """A cog housing Guild management/utility commands."""
+
+    def __init__(self, bot):
+        super().__init__(bot)
+        self.started_timers = False
+        self.timers = {}
+        if os.path.isfile(timezone_file):
+            DOZER_LOGGER.info("Loaded timezone configurations")
+            with open(timezone_file) as f:
+                self.timezones = json.load(f)
+        else:
+            DOZER_LOGGER.error("Unable to load timezone configurations")
+            self.timezones = {"abbreviations": []}
```

Comment:
Why is this dict under the `abbreviations` key? Why not just have the json file be a bare dict? At minimum, if this is a workaround for something, this should be turned into a constant.